project/src/pages/StudentDashboard/IdentityVerification.tsx

```
@@ -0,0 +1,499 @@
+
+              {loadingTerms ? (
+                <div className="flex items-center justify-center py-8">
+                  <div className="w-8 h-8 border-4 border-blue-200 border-t-blue-600 rounded-full animate-spin"></div>
+                </div>
+              ) : activeTerm ? (
+                <>
+                  <div
+                    ref={termsContentRef}
+                    onScroll={handleTermsScroll}
+                    className="max-h-[600px] overflow-y-auto prose prose-base max-w-none prose-headings:text-gray-900 prose-p:text-gray-600 prose-headings:text-xl prose-headings:font-bold prose-p:text-base prose-p:leading-relaxed mb-4 border border-slate-200 rounded-lg p-6 bg-slate-50"
+                    dangerouslySetInnerHTML={{ __html: activeTerm.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)